ivi-layermanagement-examples/EGLWLInputEventExample/src/WLContext.cpp

```diff
@@ -239,9 +235,6 @@ WLContext::DestroyWLContext()
     if (m_wlCursorTheme)
         wl_cursor_theme_destroy(m_wlCursorTheme);
 
-    if (m_wlCursor)
-        free(m_wlCursor);
```

Comment:
please check where m_wlCursor is freed now?

Reply:
Hello @efriedrich,

The wl_cursor_theme_load will allocate for wl_cursor_theme and wl_cursor. the m_wlCursor is assigned to theme->cursor. 
The wl_cursor_theme_destroy() will free wl_cursor and wl_cursor_theme. Don't need to free the m_wlCursor in application.

Look the image to get more clear, thanks.
![cursor_cycle](https://user-images.githubusercontent.com/113493212/204209654-87e5babf-122c-4cbc-a78f-6b7a70c23c33.png)